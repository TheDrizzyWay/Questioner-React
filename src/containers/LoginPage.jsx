import React from 'react';
import { Link } from 'react-router-dom';
import Header from '../components/Header';
import LoginForm from '../components/LoginForm';
import Footer from '../components/Footer';
import '../assets/styles/signup.scss';

const SignUpPage = () => {
    return (
        <div className="spg">
            <Header children="Questioner" />
            <LoginForm />
            <span>Don't have an account? Sign up using the link below.</span>
            <Link to="/signup">SIGN UP</Link>
            <Footer />
        </div>
    );
};

export default SignUpPage;
