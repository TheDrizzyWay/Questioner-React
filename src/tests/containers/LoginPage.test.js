import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../containers/LoginPage';

let wrapper;

describe('<LoginPage />', () => {
    it('Should render component successfully', () => {
        wrapper = shallow(<LoginPage />);
        expect(wrapper.length).toEqual(1);
    });

    test('Should maintain existing snapshot', () => {
        wrapper = shallow(<LoginPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
