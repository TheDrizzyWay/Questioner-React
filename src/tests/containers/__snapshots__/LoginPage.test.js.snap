// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginPage /> Should maintain existing snapshot 1`] = `ShallowWrapper {}`;
